examples/oic.r.sacl-example2.json

```diff
@@ -29,35 +29,7 @@
           }
```

Comment:
This example is intended to apply to the sacl resource - which contains a list of ACEs. (not ACE2s)